base/bitarray.jl

```diff
@@ -1296,17 +1296,12 @@ function (&)(B::BitArray, x::Bool)
 end
 (&)(x::Bool, B::BitArray) = B & x
 
-function (|)(B::BitArray, x::Bool)
-    x ? trues(size(B)) : copy(B)
```

Comment:
Please see the discussion in #17623, specifically https://github.com/JuliaLang/julia/pull/17623#issuecomment-266635187, https://github.com/JuliaLang/julia/pull/17623#issuecomment-268695364, and https://github.com/JuliaLang/julia/pull/17623#issuecomment-268703295. Best!